### ISSUE TITLE
Cairo Shell and blind accessibility for the blin d

### ISSUE THREAD

**Issue by Nuno69** [cairoshell/cairoshell#460]
Cairo Shell  is not accessible for the blind people. Some things are read by a screenreading software, yet more of them are not read completely.
Let's take the desktop as an example. When I navigate to the desktop with my screenreader I get a message saying "window", which means that the program registered that the desktop got focused, yet the icons are not read completely.
Some things cannot be accesed by the tab navigation as well. One posssibility is to use the TabOrder property to make the tab navigation.
Settings are not read completely as well. The tabs can be read and navigated, yet the settings are not read.

**Comment by dremin** [comment]
Thanks for the report! This is definitely something I want to improve, so we will track that work here.

**Comment by Nuno69** [comment]
Thank you. If you need any help or consultancy, please ask
